assets/pages/contributions-landing/new_components/ContributionAmount.jsx

```diff
@@ -0,0 +1,98 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { amounts, type Amount, type Contrib } from 'helpers/contributions';
+import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
+
+import { type CountryMetaData } from '../contributionsLandingMetadata';
+
+// ----- Types ----- //
+
+type PropTypes = {
+  countryGroupDetails: CountryMetaData,
+  countryGroupId: CountryGroupId,
+  contributionType: Contrib,
+  amount: number,
+  otherAmount: ?number,
+};
+
+// ----- Render ----- //
+
+const formatAmount = (countryGroupDetails: CountryMetaData, amount: Amount, verbose: boolean) => (verbose ?
+  `${amount.value} ${countryGroupDetails.currency.name}` :
+  `${countryGroupDetails.currency.symbol}${amount.value}`);
+
+const renderAmount = (countryGroupDetails: CountryMetaData) => (amount: Amount, i) => (
+  <li className="form__radio-group__item">
+    <input
+      id={`contributionAmount-${amount.value}`}
+      className="form__radio-group__input"
+      type="radio"
+      name="contributionAmount"
+      value={amount.value}
+      checked={i === 0}
+    />
+    <label htmlFor={`contributionAmount-${amount.value}`} className="form__radio-group__label" aria-label={formatAmount(countryGroupDetails, amount, true)}>
+      {formatAmount(countryGroupDetails, amount, false)}
+    </label>
+  </li>
+);
+
+
+function ContributionAmount(props: PropTypes) {
+  return (
+    <fieldset className="form__radio-group form__radio-group--pills form__radio-group--contribution-amount">
```

Comment:
We have [a helper function](https://github.com/guardian/support-frontend/blob/master/assets/helpers/utilities.js#L23) that you may find useful for modifiers:

```jsx
className={classNameWithModifiers('form__radio-group', ['pills', 'contribution-amount'])};
```